optimizely/client/client_test.go

```diff
@@ -1461,11 +1547,16 @@ func (s *ClientTestSuiteAB) TestActivate() {
 	}
 
 	variationKey, err := testClient.Activate("test_exp_1", testUserContext)
+	variationKey1, err1 := testClient.Activate("test_exp_2", testUserContext)
```

Comment:
nit: the enumeration is confusing. Should change the previous one to variationKey1 and set this new one as variationKey2

Reply:
addressed